pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -247,9 +250,8 @@ func TestToKubeContainerStatusWithResources(t *testing.T) {
 	)
 
 	for desc, test := range map[string]struct {
-		input         *runtimeapi.ContainerStatus
-		expected      *kubecontainer.Status
-		skipOnWindows bool
+		input    *runtimeapi.ContainerStatus
+		expected *kubecontainer.Status
 	}{
 		"container reporting cpu and memory": {
```

Comment:
If we're skipping this test on windows, can we remove the windows conditional when setting the resources? (`if goruntime.GOOS == "windows"` below)